tests/Dfc.CourseDirectory.WebV2.Tests/TestViews/SharedViews/PagerTests.cs

```diff
@@ -0,0 +1,135 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using AngleSharp.Dom;
+using Microsoft.AspNetCore.Mvc;
+using Xunit;
+
+namespace Dfc.CourseDirectory.WebV2.Tests.TestViews.SharedViews
+{
+    public class PagerTests : MvcTestBase
+    {
+        public PagerTests(CourseDirectoryApplicationFactory factory)
+          : base(factory)
+        {
+        }
+
+        [Fact]
+        public async Task RendersPage()
+        {
+            // Arrange
+
+            // Act
+            var response = await HttpClient.GetAsync("PagerContext?currentPage=6&totalPages=10&getPageUrl=1");
+
+            // Assert
+            var doc = await response.GetDocument();
+            Assert.NotNull(doc.GetElementsByClassName("page-numbers-container pagination-container"));
+        }
+
+        [Fact]
+        public async Task Pager_DoNotShowPreviousPageLinkIfCurrentPageisFirstPage()
+        {
+            // Arrange
+
+            // Act
+            var response = await HttpClient.GetAsync("PagerContext?currentPage=1&totalPages=10&getPageUrl=1");
+
+            // Assert
+            var doc = await response.GetDocument();
+            Assert.Null(doc.GetElementById("pttcd-pager__PreviousPage"));
+        }
+
+        [Fact]
+        public async Task Pager_ShowPreviousPageLinkIfCurrentPageisNumberFive()
+        {
+            // Arrange
+
+            // Act
+            var response = await HttpClient.GetAsync("PagerContext?currentPage=5&totalPages=10&getPageUrl=1");
+
+            // Assert
+            var doc = await response.GetDocument();
+            Assert.NotNull(doc.GetElementById("pttcd-pager__PreviousPage"));
+        }
+        [Fact]
+        public async Task Pager_VerifyEllipsisAreAddedWhenCurrentPageNumberisFour()
+        {
+            // Arrange
+
+            // Act
+            var response = await HttpClient.GetAsync("PagerContext?currentPage=4&totalPages=10&getPageUrl=1");
+
+            // Assert
+            var doc = await response.GetDocument();
+            Assert.NotNull(doc.GetElementsByClassName("pttcd-c-Pager__Ellipsis__First"));
+            Assert.NotNull(doc.GetElementsByClassName("pttcd-c-Pager__Ellipsis__Second"));
```

Comment:
This `Assert` shouldn't be here based on test name

Reply:
done